packages/core/src/Combobox/ComboboxRoot.vue

```diff
@@ -22,6 +22,7 @@ type ComboboxRootContext<T> = {
   parentElement: Ref<HTMLElement | undefined>
   resetSearchTermOnSelect: Ref<boolean>
   onResetSearchTerm: EventHookOn
+  tiggerResetSearchTerm: EventHookTrigger
```

Comment:
let's fix the typo here

```suggestion
  triggerResetSearchTerm: EventHookTrigger
```

Reply:
Fixed in https://github.com/unovue/reka-ui/pull/1831/commits/e6b4bf378eac2b504ec84eb1c308f5fad929349e